content/docs/command-reference/checkout.md

```diff
@@ -69,16 +69,18 @@ be pulled from remote storage using `dvc pull`.
 
 ## Options
 
+- `--summary` - show summary of the changes.
```

Comment:
I noticed checkout now has this option so I'm adding it. But this description isn't great, where can I find more details? I'm guessing @efiop may know. I.e. summary of what changes? Thanks!

Reply:
--summary is covered in the Saugat's PR that we have been trying to merge for while

Reply:
Ah yes I see there's a note about it in https://github.com/iterative/dvc.org/pull/1054#pullrequestreview-391199946. OK.

But in this PR I also changed the order of the options so I say let's keep it here for now and we'll have to solve a little conflict later.